java/system-test/system-test-dsl/src/test/java/sleeper/systemtest/dsl/testutil/drivers/InMemoryCompaction.java

```diff
@@ -155,8 +164,16 @@ private void finishTasks() {
 
     private RecordsProcessedSummary compact(CompactionJob job, TableProperties tableProperties, StateStore stateStore, String taskId) {
```

Comment:
It would be nice to break this method up a bit, it's pretty long.

Reply:
Thanks, I've split out the creation of iterators and updating the data/sketch stores into a new method